language/move-vm/types/src/native_functions/lcs.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    loaded_data::types::Type,
+    native_functions::dispatch::{native_gas, NativeResult},
+    values::{values_impl::Reference, Value},
+};
+use libra_types::vm_error::{StatusCode, VMStatus};
+use std::collections::VecDeque;
+use vm::{
+    errors::VMResult,
+    gas_schedule::{CostTable, NativeCostIndex},
+};
+
+/// Failure in LCS deserialization
+const LCS_SERIALIZATION_FAILURE: u64 = 0x01C5;
```

Comment:
Could this possibly be moved to `vm_errors` in `types`? Specifically there's a module at the bottom defining sub statuses.

Reply:
I was copying some code from `Signature` here--we should probably move that to `vm_errors` as well.